rust/test/time/basic.rs

```diff
@@ -3,6 +3,7 @@ use test_tools::exposed::*;
 
 tests_impls!
 {
+  #[ cfg( feature = "time_now" ) ]
```

Comment:
Please, give me the answer. 

Does the module `wtools` and `time_tools` works similar with given features?